### DISCUSSION TITLE
(make) failed to load external entity docbook.xsl

### DISCUSSION THREAD

**Discussion by forgottosave** [ostreedev/ostree#3210] [Q&A] [answered]
Under fedora 39, `make` sadly results in the following error message for me:

```
I/O error : Attempt to load network entity http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"
cannot parse http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
make[2]: *** [Makefile:10529: man/ostree.1] Error 4
make[2]: Leaving directory '/home/timon/Workdir/ostree'
make[1]: *** [Makefile:8254: all-recursive] Error 1
make[1]: Leaving directory '/home/timon/Workdir/ostree'
make: *** [Makefile:3338: all] Error 2
```
Do you know how this could be resolved?

**Comment by forgottosave** [comment]
I now just manually removed all manpage stuff from the Makefile and it worked. But I would still appreciate some help, not to use this dirty fix :)

**Reply by cgwalters** [reply]
In general, try using the distribution build tools to get build dependencies for things that are packaged.  For example, `dnf builddep ostree` should get you what you need.

**Reply by dbnicholson** [reply, marked as answer]
That implies that you don't have the DocBook XSL stylesheets installed. On Fedora I think the package is `docbook-style-xsl`.

**Reply by forgottosave** [reply]
Thank you so much, I got it to work now :)
